scripts_staging/linux_CheckCPUUsage.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
For better compatibility I think it was agreed we were going to try to use this:
`#!/usr/bin/env bash`

Reply:
Copy that, I'll update both of my scripts with this change